dealership/contracts.py

```diff
@@ -1,12 +1,56 @@
+from .customers import Customer, Employee
+from .vehicles import Car, Truck, Motorcycle
+
+
 class Contract(object):
-    pass
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
 
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        super(BuyContract, self).__init__(vehicle, customer)
+        self.monthly_payments = monthly_payments
+    
+    def total_value(self):
+        I = 0
```

Comment:
I did something similar in my solution. You could move this statement into the Contract class as a placeholder for the interest rate multiplier and define the multiplier constants within the Car, Motorcycle, and Truck classes in order to eliminate the if else statements.